src/core/background/scrobbler/listenbrainz-scrobbler.js

```diff
@@ -284,6 +284,14 @@ define((require) => {
 				trackMeta.additional_info.release_artist_name = song.getAlbumArtist();
 			}
 
+			if (song.getUniqueId() && song.metadata.label === 'Spotify') {
+				trackMeta.additional_info.spotify_id = song.getUniqueId();
```

Comment:
This would be something like

```
spotify:track:0VfQeEvwYx4NXuGXkbWgUb
```

@MonkeyDo is that correct? 

The spec outlines

>  The Spotify track URL associated with this recording. e.g.: http://open.spotify.com/track/1rrgWMXGCGHru5bIRxGFV0

Reply:
Indeed the `spotify_id` and the `origin_url` fields should both be a fully-qualified rather than the `spotify:track:…` URN